### ISSUE TITLE
"Reached too many edge duplications of 90% for 4 iterations. breaking" error

### ISSUE THREAD

**Issue by oguzhanaygn** [bridgecrewio/checkov#1756]
Hi,
I'm using checkov for the first time. when I run it, it gives 
```Reached too many edge duplications of 90% for 4 iterations. breaking.``` 
Error and as far as I understood, it has a limitation and because of this, my infra checking has not completed. Question is, how can increase these limitations? Is there a possible way to do it? I have almost 200 resources created via Terraform.

Any help will be highly appreaciated.
Thank you

**Comment by gruebel** [comment]
A couple of more improvements were done in this area and should hopefully unblock you to use `checkov`. Feel free to open a new issue, if you still have issues.
